helm/charts/hpe-csi-driver/README.md

```diff
@@ -39,6 +39,8 @@ The following table lists the configurable parameters of the HPE-CSI chart and t
 
 It's recommended to create a [values.yaml](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver) file from the corresponding release of the chart and edit it to fit the environment the chart is being deployed to. Download and edit [a sample file](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver).
 
+> Note: Set `storageClass.create` parameter to `false` for backendType `primera3par`.
+
```

Comment:
Can you move this to directly under the table and put it this way:

**Note:** The provided `StorageClass` is currently not compatible with `backendType: primera3par`. How to setup a `StorageClass` for HPE 3PAR and Primera is documented on [SCOD](https://scod.hpedev.io/container_storage_provider/hpe_3par_primera/).

Reply:
Done.